multi-agent/run/scripts/migrate_builtin_system.py

```
@@ -0,0 +1,262 @@
+import argparse
+import logging
+from datetime import datetime, timezone
+from uuid import uuid4
+
+import pymongo
+
+logging.basicConfig(level=logging.INFO, format="%(levelname)s: %(message)s")
+logger = logging.getLogger(__name__)
+
+
+def run_migration(db_name: str, mongodb_ip: str, mongodb_port: str, dry_run: bool, reverse: bool = False):
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)